features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js

```diff
@@ -169,7 +171,7 @@ var backendServiceInvoker = function () {
         var wsRequest = new ws.WSRequest();
         var options = new Array();
         if (IS_OAUTH_ENABLED) {
-            var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken;
+            var accessToken = getAccessToken();
```

Comment:
Ack, Moved to a constant in commit https://github.com/wso2/carbon-device-mgt/commit/3e537e2d841ca3c7946cd34e9da96f261e0ac94e1